templates/zerver/integrations.html

```diff
@@ -1790,18 +1797,47 @@
 
       <p>Finally, set up a webhook in your <code>.travis.yml</code> file:</p>
 
-<pre>
-notifications:
-  webhooks:
-    - https://zulip.example.com/api/v1/external/travis?stream=travis&amp;topic=build-status&amp;api_key=abcdefgh
-</pre>
+      <pre>
+      notifications:
+        webhooks:
+          - https://zulip.example.com/api/v1/external/travis?stream=travis&amp;topic=build-status&amp;api_key=abcdefgh
+      </pre>
```

Comment:
Can you use spaces rather than tabs here?  Zulip in general uses spaces for all whitespace since they are consistent across editors.